### PR TITLE
Add hyperv management tools to Windows image

### PR DESCRIPTION
**Pull request by n1hility**
Update the feature to pull in management tools to support the ability to use powershell scripts and the hyper-v management tools for debugging

### CODE REVIEW

**Comment by n1hility** [comment]
PTAL @cevich 

(Also note the timebomb extension in here)

**Comment by n1hility** [comment]
I'm going to update this PR to add a storage size increase based on recent failures @baude observed

**Comment by n1hility** [comment]
PTAL one more time @cevich or @edsantiago I added a small storage change so just need to make sure you guys are cool with it

**Comment by edsantiago** [comment]
Versions:

|*Distro*|*base*|*aardvark-dns*|*buildah*|*conmon*|
|-----|----|----|----|----|
|*debian*| 13 | 1.4.0-5 | 1.32.2+ds1-1 | 2.1.6+ds1-1 |
|*prior-fedora*| 38-1.6 | - | 1.33.2-1 | 2:2.1.7-2 |
|*fedora*| 39-1.5 | 1.9.0-1 | 1.33.2-1 | 2:2.1.8-2 |
|*fedora-aws*| - | 1.8.0-1 | 1.33.2-1 | 2:2.1.8-2 |
|*rawhide*| 40-0 | 1.9.0-1 | 1.33.2-1 | 2:2.1.8-2 |

|*Distro*|*container-selinux*|*containers-common*|*criu*|*crun*|
|-----|----|----|----|----|
|*debian*| - | - | 3.17.1-3 | 1.12-1 |
|*prior-fedora*| 2:2.224.0-1 | 4:1-89 | 3.18-1 | 1.12-1 |
|*fedora*| 2:2.226.0-1 | 4:1-95 | 3.19-2 | 1.12-1 |
|*fedora-aws*| 2:2.224.0-1 | 4:1-95 | 3.18-3 | 1.12-1 |
|*rawhide*| 2:2.226.0-1 | 4:1-100 | 3.19-2 | 1.12-1 |

|*Distro*|*golang*|*netavark*|*passt*|*podman*|
|-----|----|----|----|----|
|*debian*| 2:1.21~2 | 1.4.0-4 | 0.0~git20231107.74e6f48-1 | 4.7.2+ds1-2 |
|*prior-fedora*| 1.20.11-1 | - | 0^20231107.g56d9f6d-1 | 5:4.7.2-1 |
|*fedora*| 1.21.4-1 | 1.9.0-1 | - | 5:4.8.1-1 |
|*fedora-aws*| 1.21.4-1 | 1.8.0-2 | 0^20231119.g4f1709d-1 | 5:4.8.0-1 |
|*rawhide*| 1.21.5-1 | 1.9.0-1 | 0^20231204.gb86afe3-1 | 5:4.8.0-1 |

|*Distro*|*runc*|*skopeo*|*slirp4netns*|*systemd-container*|
|-----|----|----|----|----|
|*debian*| 1.1.10+ds1-1 | 1.13.3+ds1-2 | 1.2.1-1 | 255~rc4-2 |
|*prior-fedora*| 2:1.1.8-1 | 1:1.14.0-1 | 1.2.2-1 | 253.12-1 |
|*fedora*| 2:1.1.8-1 | 1:1.14.0-1 | 1.2.2-1 | 254.7-1 |
|*fedora-aws*| 2:1.1.8-1 | 1:1.14.0-1 | 1.2.2-1 | 254.7-1 |
|*rawhide*| 2:1.1.9-1 | 1:1.14.0-1 | 1.2.2-1 | 255~rc4-3 |

**Comment by edsantiago** [comment]
Eh, we can do the podman testing on our end.

/lgtm

Thanks @n1hility

File: win_images/win-server-wsl.yml [hunk 1]
```diff
@@ -34,7 +34,11 @@ builders:
     security_group_id: "sg-042c75677872ef81c"
     ami_name: &ami_name '{{build_name}}-c{{user `IMG_SFX`}}'
     ami_description: 'Built in https://cirrus-ci.com/task/{{user `CIRRUS_TASK_ID`}}'
-
+    launch_block_device_mappings:
+      - device_name: '/dev/sda1'
+        volume_size: 200
+        volume_type: 'gp2'
+        delete_on_termination: true
```
Comment (cevich) [inline win_images/win-server-wsl.yml:41]: This LGTM, though from what I've seen in Mac-land, default `gp2` and `gp3` may be "too slow" for podman-machine testing.  IIRC, they only provision 150 IOPS.  Going up to 'gp3' + 600 IOPS is still not great, but it's a big improvement for a small increase in cost.

If you have a chance to re-push/build, would you mind switching to `gp3` and setting `iops: 600`?

**Comment by cevich** [comment]
Merging since this passes podman CI.  Let's do the storage-speed bump in a followup.

**Pull request merged**